src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/PropertyGroupIntrinsicTask.cs

```diff
@@ -111,6 +117,51 @@ internal override void ExecuteTask(Lookup lookup)
             }
         }
 
+        /// <summary>
+        /// Logs property assignment information during target execution, providing detailed tracking of property value changes.
+        /// </summary>
+        /// <param name="property">The property instance being assigned or modified.</param>
+        /// <param name="evaluatedValue">The new evaluated value of the property.</param>
+        private void LogPropertyInTargetAssignment(ProjectPropertyGroupTaskPropertyInstance property, string evaluatedValue)
+        {
+            if (_propertyTrackingSettings == 0 || !ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_10))
```

Comment:
This change now is targetting 17.14 - so `ChangeWaves.Wave17_10` technically won't be already available at that time.

So either change to 17_14, or just remove completely